common/src/stack/command/stack/commands/set/host/power/__init__.py

```diff
@@ -25,79 +27,106 @@ class Command(stack.commands.set.host.command):
 	</param>
 
 	<param type='boolean' name='debug' optional='1'>
-	Print debug output from the ipmitool command.
+	Print debug output from the command. For ipmi capable hosts, prints
+	the output from ipmitool.
+	</param>
+
+	<param type='string' name='force' optional='1'>
+	Force a method for setting power on a host.
```

Comment:
"Force", I believe, is normally used in other commands to force it to go on even if some validation fails. It might be better to name this something else, maybe "method" or "via"?

Also consider listing the accepted values for this if the number is relatively small.

Reply:
I wasn't entirely sure on the parameter name, I'm thinking use-method would be better. As far as listing the methods, I'm not sure if another flag would would be the best option that prints the available power implementations.